src/Caching/SqlServer/src/Microsoft.Extensions.Caching.SqlServer.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Distributed cache implementation of Microsoft.Extensions.Caching.Distributed.IDistributedCache using Microsoft SQL Server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

Comment:
@JunTaoLuo looks like this was from extensions - does the cross-targeting here make sense?

Reply:
I don't see why not. Other extension packages like [Primitives](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.csproj#L3) are also cross-targeted.

Reply:
Do it, do it, do it 😁